<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>551</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tabDroplets">
     <attribute name="title">
      <string>Droplets</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>85</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Get Droplets</string>
      </property>
     </widget>
     <widget class="QTreeWidget" name="treeDroplets">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>661</width>
        <height>191</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>661</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>661</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <attribute name="headerCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="headerDefaultSectionSize">
       <number>200</number>
      </attribute>
      <attribute name="headerMinimumSectionSize">
       <number>16</number>
      </attribute>
      <attribute name="headerStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Droplet</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>IP address</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Region</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tabSettings">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QPushButton" name="btnSaveApiKey">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>85</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Save API Key</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>23</x>
        <y>32</y>
        <width>581</width>
        <height>29</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>API Key: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtApiKey"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Alt+Q</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
   <property name="toolTip">
    <string>Start droplet</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop droplet</string>
   </property>
  </action>
  <action name="actionReboot">
   <property name="text">
    <string>Reboot</string>
   </property>
   <property name="toolTip">
    <string>Reboot droplet</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
